proto/internalapi/sensor/network_connection_iservice.proto

```diff
@@ -48,7 +66,7 @@ message IPNetworkList {
 
 // A Sensor service that allows Collector to send NetworkConnectionInfo messages
 service NetworkConnectionInfoService {
-  // Note: the response is a stream due to a bug in the C++ GRPC client library. The server is not expected to
-  // send anything via this stream.
```

Comment:
Collector sends `NetworkConnectionInfoMessage` messages so this comment does not appear to be correct.